File: MPI/mpicheck/tests/functions/2.0/MPI_File_read_ordered_begin.f90

        
        program main
        use mpi
        
        INTEGER var_0
       TYPE(INTEGER) var_1(10)
       INTEGER var_2
       INTEGER var_3
       INTEGER var_4
        call mpi_file_read_ordered_begin(var_0, var_1, var_2, var_3, var_4)
       call pmpi_file_read_ordered_begin(var_0, var_1, var_2, var_3, var_4)
        end program main
    

File: MPI/mpicheck/tests/functions/2.0/MPI_File_read_ordered_begin.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_File var_0;
    void *var_1;
    int var_2;
    MPI_Datatype var_3;
    int ret;
    /* calls */
    ret = MPI_File_read_ordered_begin(var_0, var_1, var_2, var_3);
    ret = PMPI_File_read_ordered_begin(var_0, var_1, var_2, var_3);
    return 0;
}
